pkg/kudoctl/cmd/test.go

```diff
@@ -104,6 +108,27 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 				options.StartControlPlane = startControlPlane
 			}
 
+			if isSet(flags, "start-kind") {
```

Comment:
With all of these extra additions, what do you need from isSet/pflag that Cobra isn't doing for you? Do we need to evaluate a new CLI parser?

Reply:
It’s not a bad idea, as pflags seems pretty limited. The isSet method is so I can tell the difference between a falsy value and a value that has not been specified. Some other things I’m doing:

* loading config from a file and overriding with command line flags
* mutual exclusivity
* a “one of this set of flags must be set” semantic